docs/painless/painless-contexts/painless-ingest-processor-context.asciidoc

```diff
@@ -38,4 +38,145 @@ void::
 
 *API*
 
-The standard <<painless-api-reference, Painless API>> is available.
\ No newline at end of file
+The standard <<painless-api-reference, Painless API>> is available.
+
+*Example*
+
+To use this example first follow the steps outlined in
+<<painless-context-examples, context examples>>.
+
```

Comment:
I'd be more explicit... "To run this example, you need to <<painless-context-examples, ingest the sample data>>.

Reply:
Fixed.